sbpy/data/core.py

```diff
@@ -635,11 +638,12 @@ def __getitem__(self, ident):
         elif isinstance(ident, str):
             self = self._convert_columns(ident)
             ident = self._translate_columns(ident)[0]
+            return self._table[ident]
         elif isinstance(ident, int):
-            return self.from_table(self._table[ident])
+            pass
```

Comment:
Same thing for int ident.

Reply:
... and the same answer. Thanks for checking!